stubs/pyinstaller/PyInstaller/lib/modulegraph/modulegraph.pyi

```diff
@@ -2,15 +2,16 @@
 # We reference the vendored package rather than depending on the original untyped module.
 # Anything not referenced in the PyInstaller stubs doesn't need to be added here.
 
-from typing import Any, Protocol
+from _typeshed import Incomplete
+from typing import Protocol
 
 class _SupportsGraphident(Protocol):
     graphident: str
 
 # code, filename and packagepath are always initialized to None. But they can be given a value later.
 class Node:
     # Compiled code. See stdlib.builtins.compile
-    code: Any | None
+    code: Incomplete | None
```

Comment:
You only added this one recently and, from the comment, it looks like it's meant to be `Any` ;)

Reply:
Definitely, I don't see anything else obvious after manually reviewing as well.

Reply:
I haven't checked the code but it looks like this should just be `types.CodeType`.

Reply:
This PR no longer touches this line, so I think that can be deferred to another PR

Reply:
It's the return of `stdlib/builtins.pyi > compile`, which is currently typed as `Any`, but also has this comment:
> # TODO: `compile` has a more precise return type in reality; work on a way of expressing that?

Reply:
- #9567

Reply:
Right, I'd be surprised if PyInstaller is using PyCF_ONLY_AST here.

Reply:
(technically it's modulegraph, pyinstaller vendors it) It is using PyCF_ONLY_AST  as well, but it's the non-ast call that is stored.